Source/ASDisplayNode.h

```diff
@@ -209,7 +209,7 @@ ASDK_EXTERN NSInteger const ASDefaultDrawingPriority;
  * @warning The first access to it must be on the main thread, and should only be used on the main thread thereafter as 
  * well.
  */
-@property (readonly) UIView *view;
+@property (readonly) NS_SWIFT_UI_ACTOR UIView *view;
```

Comment:
The documentation explains this: Only the first access (factory) is required to be in the main thread. Subsequent accesses may be called from background threads as both the view and layers may be passed around nodes.

Reply:
I'm not sure, do you mean the comment is wrong ? reading it I explicitly understand it as main thread only

Reply:
> The first access to it must be on the main thread, and should only be used on the main thread thereafter as well.

The latter part refers to the view itself (which is a UIKit rule more than Textures). If you check the internal implementation of `- (UIView *)view` on `ASDisplaynode.mm`, the main thread assertion is only when the view needs to be created. The same is true for `- (CALayer *)layer`. Texture needs to access this internally even if the node itself is meant for background use.